src/implementations/DopplerLegacyClientImpl.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  DopplerLegacyClient,
+  DopplerLegacyUserData,
+} from "../abstractions/doppler-legacy-client";
+import { ResultWithoutExpectedErrors } from "../abstractions/common/result-types";
+import { AxiosResponse, AxiosStatic } from "axios";
+import { AppConfiguration } from "../abstractions";
+
+export class DopplerLegacyClientImpl implements DopplerLegacyClient {
+  private instance;
```

Comment:
What do you think about naming it `axios`?

```suggestion
  private axios: Axios;
```

Reply:
I will change 👍